addons/rating/models/rating.py

```diff
@@ -133,6 +134,44 @@ def action_open_rated_object(self):
         }
 
 
+class RatingParentMixin(models.AbstractModel):
```

Comment:
Linting: we discussed to have another file with this new mixin + the other existing one.